astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
+The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.
+</Issue>
+
+<Issue issue="2893,2472">
+FusionAuth is not honoring the `rememberDevice` setting set by a user when performing a federated login. This setting tells FusionAuth whether it should establish an SSO session for the user. FusionAuth will look for this value in the following order: 
+1. The selection previously made by the user (stored in a cookie)
+2. The checkbox on interactive login pages
```

Comment:
This one should be first. If it was an interactive login, and there was a submitted form field, that will take precedence over the cookie. The cookie is used in non-interactive workflows (e.g. where the user is immediately redirected because of `idp_hint`)